_data/i18n/en.yml

```diff
@@ -271,3 +271,35 @@ ci_semaphore: Semaphore
 search_placeholder: Search packages (i.e. babel, webpack, react…)
 search_by_algolia: Search by Algolia
 search_by_read_more: read how it works
+
+downloads_in_last_30_days: downloads in the last 30 days
+npm_page_for: NPM page for
```

Comment:
These should have placeholders (eg. `NPM page for {name}`). Some languages have different sentence order, so simply appending the name won't work in all cases.